src/app/shared/refresh-organization/refresh-organization.component.html

```diff
@@ -14,5 +14,5 @@
   ~    limitations under the License.
   -->
 
-<button tooltip="Refresh organization" placement="right" type="button" class="btn btn-primary btn-xs glyphicon glyphicon-refresh" (click)="refreshOrganization()" [disabled]="toDisable()">
+<button tooltip="Refresh organization" placement="right" style="margin-right:5px;" type="button" class="btn btn-primary btn-xs glyphicon glyphicon-refresh" (click)="refreshOrganization()" [disabled]="toDisable()">
```

Comment:
We've generally avoided inline styles.  Rename to .mr-1 or .my-2, etc for when we ever switch to https://v4-alpha.getbootstrap.com/utilities/spacing/